desktop/apps/categories2/stylesheets/index.styl

```diff
@@ -1,22 +1,23 @@
 .categories2-app-container {
-	position relative
-	padding-bottom 3em
+	display flex
+	flex 0 1 auto
+	flex-flow row wrap
+	justify-content flex-end
 }
 
 .categories2-sidebar {
-	height 100%
+	flex 1
 	font-size 14px
 	color #333
-	left 0
-	padding 1em
-	position absolute
-	top 30px
+	padding 2em 1em
 	text-transform uppercase
-	width 300px
+	ul {
+		padding-top 2em
+	}
```

Comment:
I like the idea, for many [reasons](https://github.com/artsy/rosalind/pull/70).

I think we should be conservative about adding new dependencies to Force but this certainly seems like a win. 

And sooner is better than later — I've already found it challenging to retrofit an existing app to use styled components.